File: src/main/java/rbq/wtf/lycoris/client/wrapper/wrappers/annotation/repeat/WrapFields.java
package rbq.wtf.lycoris.client.wrapper.wrappers.annotation.repeat;

import rbq.wtf.lycoris.client.wrapper.wrappers.annotation.WrapField;

import java.lang.annotation.*;

@Documented
@Target(ElementType.FIELD)
@Retention(RetentionPolicy.RUNTIME)
/**
 * Auto setup field
 */
public @interface WrapFields {
    WrapField[] value();
}

File: src/main/java/rbq/wtf/lycoris/client/wrapper/wrappers/annotation/repeat/WrapFields.kt
package rbq.wtf.lycoris.client.wrapper.wrappers.annotation.repeat

import rbq.wtf.lycoris.client.wrapper.wrappers.annotation.WrapField

@MustBeDocumented
@Target(AnnotationTarget.FIELD)
@kotlin.annotation.Retention(AnnotationRetention.RUNTIME)
/**
 * Auto setup field
 */
annotation class WrapFields(vararg val value: WrapField)